openml/runs/functions.py

```diff
@@ -391,26 +395,103 @@ def _run_task_get_arffcontent(
     # TODO use different iterator to only provide a single iterator (less
     # methods, less maintenance, less confusion)
     num_reps, num_folds, num_samples = task.get_split_dimensions()
+    classes = None
 
+    n_fit = 0
     for rep_no in range(num_reps):
```

Comment:
maybe replace this with `itertools.product(num_reps, num_folds, num_samples)` so we get rid of two levels of indentation and can maybe spare some ugly line-breaks below.

Reply:
Done